File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/RegionModel.ts
import { ClientModel } from "./clientModel";
import { GovernerateModel } from "./governerateModel";

export interface RegionModel {
    regionId: number;
    regionNameAr: string;
    regionNameEn: string;
    regionCode: string;
    isActive: boolean | null;
    displayOrder: number | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    color: string;
    icon: string;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}

File: SofiForce.Models/Models/EntityModels/RegionModel.cs
using System;

namespace Models
{
    public class RegionModel
    {
        public int RegionId { get; set; }
        public string RegionNameAr { get; set; }
        public string RegionNameEn { get; set; }
        public string RegionCode { get; set; }
        public bool? IsActive { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
